exec/java-exec/src/test/java/org/apache/drill/exec/store/TestImplicitFileColumns.java

```diff
@@ -110,4 +111,20 @@ public void testImplicitColumnsForParquet() throws Exception {
         .go();
   }
 
+  @Test // DRILL-4733
+  public void testMultilevelParquetWithSchemaChange() throws Exception {
+    try {
+      test("alter session set `planner.enable_decimal_data_type` = true");
```

Comment:
One of the parquet files in the data set contain a column which is double. But I do not understand why drill requires us to enable the decimal type;